runelite-client/src/main/java/net/runelite/client/plugins/textrecolor/TextRecolorConfig.java

```diff
@@ -0,0 +1,589 @@
+package net.runelite.client.plugins.textrecolor;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+import java.awt.*;
+
+@ConfigGroup(
+	keyName = "textrecolor",
+	name = "Chat Text Recolor",
+	description = "Configuration for chat text recoloring"
+)
+public interface TextRecolorConfig extends Config
+{
+	@ConfigItem(
+		position = 30,
+		keyName = "__dummyTitleOpaque",
+		name = "Opaque Chat Colors",
+		description = ""
+	)
+	default long __dummyTitleOpaque()
+	{
+		return 0;
+	}
+
+	@ConfigItem(

```

Comment:
Why are you starting at 31

Reply:
I wanted to leave space for settings to go above. I have some code for text highlights that starts at 0. 30 seemed like a good number of spaces to leave.